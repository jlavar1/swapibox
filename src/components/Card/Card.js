import React from 'react';

const Card = (props) => {
  return (
    <section className='card'>
      This is a card living in the CC
    </section>
  )
}

export default Card;