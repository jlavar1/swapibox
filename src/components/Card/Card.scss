.card {
  border: 2px solid #feda4a;
  font-family: 'Poller One', cursive;
  width: 15rem;
  margin: 1rem;
  padding: 1rem;
  display: flex;
  justify-content: center;
  align-items: center;
  background: #feda4a;
  font-family: 'Poller One', cursive;
}