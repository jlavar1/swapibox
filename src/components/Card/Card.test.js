import React from 'react';
import Card from './Card';
import { shallow } from 'enzyme';

describe('Card', () => {
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(
      <Card />
    );
  });
  
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });

  it('should have the proper default state', () => {
    expect(wrapper.state()).toEqual({});
  });
})