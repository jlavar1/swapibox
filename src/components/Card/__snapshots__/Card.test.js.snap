// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot 1`] = `
<div>
  This is a card living in the CC
</div>
`;
